tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -2302,9 +2302,11 @@ public function statusPhpRespondedShouldMatch(PyStringNode $jsonExpected): void
 		$jsonExpectedDecoded = \json_decode($jsonExpected->getRaw(), true);
 		$jsonRespondedDecoded = $this->getJsonDecodedResponse();
 
-		$this->appConfigurationContext->theAdministratorGetsCapabilitiesCheckResponse();
+		$response = $this->appConfigurationContext->userGetsCapabilities($this->appConfigurationContext->getAdminUsernameForCapabilitiesCheck());
+		$this->theHTTPStatusCodeShouldBe(200, '', $response);
+		$response_xml = $this->getResponseXml($response)->data->capabilities;
```

Comment:
can we not use snake case as rest of the files are using camel case
```suggestion
		$responseXml = $this->getResponseXml($response)->data->capabilities;
```